src/connector.rs

```diff
@@ -37,14 +37,6 @@ pub enum TokenMessageData {
     Eth { address: AccountId, message: String },
 }
 
-/// Key prefix to creating key for storage
-#[derive(Clone, Copy, BorshSerialize, BorshDeserialize)]
-pub enum EthConnectorKeyPrefix {
```

Comment:
I had to move this into `storage` so that it could be accessed from tests. I also renamed it when I moved it because it's not always a prefix (sometimes it plus `KeyPrefix` is the whole key).